source/options.html

```diff
@@ -86,17 +73,15 @@
 		<p>
 			<button id="find-feature">Start process to identify feature…</button>
 		</p>
-	</div>
+	</details>
 
-	<hr>
-
-	<p>
-		<label>
-			<strong class="features-header">🔋 Features</strong>
-			<input id="filter-features" type="text" placeholder="Find features" spellcheck="false" autocomplete="off" autocapitalize="off">
-		</label>
-	</p>
-	<div class="js-features"></div>
+	<details id="features">
```

Comment:
Should the features section be open by default, given that it is the last section and likely the one you want to access anyway?

```suggestion
	<details id="features" open>
```

Reply:
Mixed feelings about his, I like that entire page is compact. Keeping it open takes that away.